playbooks/rds_monitoring_stop_server.yaml

```diff
@@ -0,0 +1,37 @@
+---
+- name: Prepare controller variables
+  import_playbook: prepare_variables.yaml
+  vars:
+    scenario_name: >
+      csm_controller
+      rds_monitoring
+
+- name: Prepare variables
+  hosts: localhost
+  vars_files:
+    - ./vars/csm_controller.yaml
+    - ./vars/rds_monitoring.yaml
+  tasks:
+    - name: Register rds_host
+      add_host:
+        name: rds_host
+        ansible_host: "{{ rds_monitoring.ecs_ip }}"
+        ansible_ssh_user: linux
+        ansible_ssh_private_key_file: "{{ csm_controller.controller_key }}"
```

Comment:
`ansible_ssh_user` and `ansible_ssh_private_key_file` should be set in inventory

Reply:
fixed